beacon-chain/simulator/service.go

```diff
@@ -43,26 +47,28 @@ type Config struct {
 // DefaultConfig options for the simulator.
 func DefaultConfig() *Config {
 	return &Config{
-		Delay:                       time.Second * 8,
+		Delay:                       time.Second * 5,
 		BlockRequestBuf:             100,
 		CrystallizedStateRequestBuf: 100,
 	}
 }
 
 // NewSimulator creates a simulator instance for a syncer to consume fake, generated blocks.
-func NewSimulator(ctx context.Context, cfg *Config, beaconp2p types.P2P, web3Service types.POWChainService, chainService types.StateFetcher) *Simulator {
+func NewSimulator(ctx context.Context, cfg *Config, beaconDB ethdb.Database, beaconp2p types.P2P, web3Service types.POWChainService, chainService types.StateFetcher) *Simulator {
```

Comment:
6 args is way too many.

Is there a reason to have so many? If this is out of scope, we can solve in another PR.